PIM/Operation/01_ManageProducts.md

```diff
@@ -0,0 +1,313 @@
+[!!Omni-Channel](../../Channels/Overview/01_General.md)   
+[!!User Interface Product list](../UserInterface/02a_List.md)   
+[!!Manage the variants](./02_ManageVariants.md)
+
+
+# Manage a product
+
+A product defines everything that can be offered and purchased on a market. All products are assigned to an attribute set and can be specified by the attributes within this set. You can create master products and [variants](./02_ManageVariants.md#create-variants), edit and copy products. To offer a product on a marketplace you have to create an offer to the product, see [Create an offer in PIM](#create-an-offer-in-PIM).
```

Comment:
```suggestion
You can create master products and [variants](./02_ManageVariants.md#create-variants), edit and copy products. 
```

Verlinkung so oder "variants" ohne Verlinkung und dann im neuen Satz, z.B.: To manage product variants, see [Manage variants](./02_ManageVariants.md).